Doc/library/asyncio-task.rst

```diff
@@ -624,10 +624,23 @@ Timeouts
     The context manager produced by :func:`asyncio.timeout` can be
     rescheduled to a different deadline and inspected.
 
-    .. class:: Timeout()
+    .. class:: Timeout(when)
 
        An :ref:`asynchronous context manager <async-context-managers>`
-       that limits time spent inside of it.
+       for cancelling overdue coroutines.
+
+       .. note::
+
+            Please use :meth:`timeout()` or :meth:`timeout_at()`
+            rather than instantiating this class directly.
+
```

Comment:
I'm not sure we really need this note, since the entire section of docs for this class are enclosed within the docs for `asyncio.timeout`

```suggestion
```